src/tests/Microsoft.PowerFx.Connectors.Tests/Microsoft.PowerFx.Connectors.Tests.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework Condition="'$(BuildTarget)' == 'netcoreapp31'">netcoreapp3.1</TargetFramework>
+		<TargetFrameworks Condition="'$(BuildTarget)' == ''">netcoreapp3.1</TargetFrameworks>
+		<IsPackable>false</IsPackable>
+		<GenerateDocumentationFile>false</GenerateDocumentationFile>
+		<Configurations>Debug;Release</Configurations>
+		<OutputType>Library</OutputType>		
+	</PropertyGroup>
+	<ItemGroup>
+	  <None Remove="Swagger\MSNWeather.json" />
+	  <None Remove="Swagger\MSNWeather_Response.json" />
+	  <None Remove="Swagger\TestOpenAPI.json" />
+	</ItemGroup>
+	<ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+        <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
>Newtonsoft.Json [](http://example.com/codeflow?start=35&length=15)

why newtonsoft and not stj?

Reply:
Actually, I don't even see this using newtonsoft anywhere

Reply:
it's an additional dependency, STJ is part of the standard for new .NET versions, and is faster.